src/Components/Form/AddFormButton.js

```diff
@@ -1,4 +1,4 @@
-import ButtonAdd from '../UI/Button/AddButton.styles';
+import ButtonAdd from '../UI/Form/Button/AddButton.styles';
 
 export default function AddFormButton({ onSetFormButton }) {
```

Comment:
When "using" your component on the `posts.js` page, there's a second prop called `formButton` - it is missing here in the prop destructuring. You can either delete the prop on the posts page or you decide to use it inside the component.

Reply:
It seems you deleted the `AddFormButton` component and decided for a more straight forward approach